File: sample_app/src/main/java/uk/co/alt236/btlescan/ui/details/recyclerview/model/HeaderItem.java
package uk.co.alt236.btlescan.ui.details.recyclerview.model;

import uk.co.alt236.btlescan.ui.common.recyclerview.RecyclerViewItem;

public class HeaderItem implements RecyclerViewItem {
    private final CharSequence mText;

    public HeaderItem(CharSequence text) {
        mText = text;
    }

    public CharSequence getText() {
        return mText;
    }
}

File: sample_app-modules/ui/src/main/java/uk/co/alt236/btlescan/app/ui/view/details/model/HeaderItem.kt
package uk.co.alt236.btlescan.app.ui.view.details.model

import uk.co.alt236.btlescan.app.ui.view.recyclerview.RecyclerViewItem

interface HeaderItem :
    DetailsScreenItems,
    RecyclerViewItem {
    val text: CharSequence
}
